app/jobs/send_complaint_job.rb

```diff
@@ -3,6 +3,9 @@ class SendComplaintJob < ApplicationJob
 
   def perform(form_builder_payload:)
     Rails.logger.warn("Working on job_id: #{job_id}")
+
+    Usecase::SpawnAttachments.new(form_builder_payload: form_builder_payload).call
```

Comment:
I thought we would "spawn" the attachments when we receive the initial `POST` request from the submitter. Eg:
1. Controller receives POST
2. Controller iterates through the attachments array and saves each one to the db
3. Controller invokes the `SendComplaintJob`, sending the complaint to Optics

Reply:
i considered that, but why do it then when we can do it async? I don't see why the API client has to wait